js/components/portal-dashboard/overlay-class-response.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Map } from "immutable";
+import ArrowIcon from "../../../img/svg-icons/arrow-icon.svg";
+
+import css from "../../../css/portal-dashboard/overlay-class-response.less";
+
+interface IProps {
+  currentQuestion?: Map<string, any>;
+}
+interface IState {
+  hideResponseArea: boolean;
+}
+
+export class ClassResponse extends React.PureComponent<IProps, IState> {
+  constructor(props: IProps) {
+    super(props);
+    this.state = {
+      hideResponseArea: false
+    };
+  }
+  render() {
+    let chevronClass = `${css.arrow}`;
+    let responseAreaClass = `${css.responseArea}`;
+
+    if(this.state.hideResponseArea) {
+      chevronClass += ` ${css.hideResponseArea}`;
+      responseAreaClass += ` ${css.hidden}`;
+    }
```

Comment:
Both of these definitions can be shortened to a single line.  Something like:
```
    const chevronClass = this.state.hideResponseArea ? `${css.arrow} ${css.hideResponseArea}` : css.arrow;
    const responseAreaClass = this.state.hideResponseArea ? `${css.responseArea} ${css.hidden}` : css.responseArea;
```

Reply:
For some reason CodeClimate complained on that one about duplication of code when I wrote it the way you have. So I changed it to the above to pass CodeClimate

Reply:
I think it's better as the single line const.  I would ignore the code climate complaint.